plugins/sinks/columbus/sink_test.go

```diff
@@ -79,7 +79,9 @@ func TestSink(t *testing.T) {
 			t.Fatal(err)
 		}
 
-		columbusSink.Sink(ctx, []models.Record{models.NewRecord(topic)})
+		if err = columbusSink.Sink(ctx, []models.Record{models.NewRecord(topic)}); err != nil {
+			return
```

Comment:
Instead of return should we assert error as `nil` since it is a test??

Reply:
Checked it only return is supporting ,neither `return err` nor `return nil`

Reply:
I was saying if we should use a return statement there and not a assert statement there. like `assert.Nil(t, err)`

Reply:
got it , changed 👍